metadata-db-app/src/cmr/metadata_db/migrations/078_add_subscription_type_to_subscription_table.clj

```diff
@@ -0,0 +1,47 @@
+(ns cmr.metadata-db.migrations.078-add-subscription-type-to-subscription-table
+  (:require
+   [cheshire.core :as json]
+   [cmr.common.util :as util]
+   [cmr.common-app.services.ingest.subscription-common :as sub-common]
+   [config.mdb-migrate-helper :as helper]))
+
+(defn result->type
+  "From a Database result, extract out the query and convert it to a usable string."
+  [result]
+  (let [metadata (-> (:metadata result)
+                     util/gzip-blob->string
+                     (as-> % (json/parse-string %, true)))
+        collection-concept-id (:CollectionConceptId metadata)
+        subscription-type (:Type metadata)]
+    (or subscription-type
+        (when collection-concept-id
+          "granule")
+        "collection")))
+
+(defn populate-new-column
+  "Pull out the Type from the matadata, then populate the new 'subscription_type' column."
+  []
+  (doseq [result (helper/query "SELECT id, concept_id, metadata FROM cmr_subscriptions")]
```

Comment:
Since the only subscriptions in the table are granule subscriptions, we can simplify this function to just run:
```
UPDATE cmr_subscriptions SET subscription_type='granule'
```